### ISSUE TITLE
[Probabilistic forecasting with TFT] Quantiles predictions values

### ISSUE THREAD

**Issue by ZineebF** [Nixtla/neuralforecast#1199]
### What happened + What you expected to happen

I used the MQLoss while training the TFT model with level=[60, 80, 90], and when I call predict to get the predictions, I found out that, sometimes, the values for TFT-lo-60 are lower than TFT-lo-80 and TFT-lo-90 and also the values of TFT-hi-60 are lower than TFT-median. Based on the level I set in the loss I am expecting to get 5th percentile (lo-90), 10th percentile (lo-80), 20th percentile (lo-60), 50th percentile (median), 80th percentile (hi-60), 90th (hi-80), and 95th (hi-90). Thus the predictions of the corresponding columns should be sorted in an ascending order. 
<img width="774" alt="Screenshot 2024-11-13 at 10 59 42" src="https://github.com/user-attachments/assets/e75bf79a-da9c-4178-8540-5619da59ee36">

### Versions / Dependencies

nueuralforecast 1.7.5
python 3.10
lightning 2.4.0

### Reproduction script

nf_model = NeuralForecast(
        models = [TFT(
            h = 36,
            input_size = 336,
            futr_exog_list = ['month','day','hour'],
            hidden_size = 32,
            n_head = 4,
            max_steps = 1000,
            learning_rate = 0.0001,
            val_check_steps = 5,
            early_stop_patience_steps = 5,
            loss = MQLoss(level=[60, 80, 90]),
            enable_progress_bar=True,
            step_size = 24)],
        freq='h',
    )

nf_model.fit(df=data, val_size = 24*7*4)

nf_model.predict(df=hist_df, futr_df=futr_df)

### Issue Severity

High: It blocks me from completing my task.

**Comment by elephaint** [comment]
This is a known feature of MQLoss; there is nothing preventing MQLoss from having non-monotonic outputs.

The solution is to choose more quantiles (e.g. 10 or 20), and sort your outputs, and recompute the quantiles.
